test_files/arrow_fn.untyped/arrow_fn.untyped.js

```diff
@@ -1 +1,5 @@
-goog.module('test_files.arrow_fn.untyped.arrow_fn.untyped');var module = module || {id: 'test_files/arrow_fn.untyped/arrow_fn.untyped.js'};var /** @type {?} */ fn3 = (a) => 12;
+goog.module('test_files.arrow_fn.untyped.arrow_fn.untyped');var module = module || {id: 'test_files/arrow_fn.untyped/arrow_fn.untyped.js'};/**
```

Comment:
this stuff was all jammed on one line to avoid changing the sourcemap offsets, right? Is there any reason we still have to do that? If there is a reason, does this change cause a problem with it?